pkg/scalers/azure/azure_eventhub_test.go

```diff
@@ -12,6 +12,7 @@ import (
 
 	"github.com/Azure/azure-storage-blob-go/azblob"
 	"github.com/go-playground/assert/v2"
+	kedav1alpha1 "github.com/kedacore/keda/v2/apis/keda/v1alpha1"
```

Comment:
Could you split internal imports in another group? I mean, please, add an empty line before this import just to split imports into "golang", "dependencies" and "internals"